certbot_dns_desec/dns_desec.py

```diff
@@ -46,7 +46,7 @@ def _setup_credentials(self):
             "credentials",
             "deSEC credentials INI file",
             {
-                # "endpoint": "URL of the deSEC API.", # TODO add support for different endpoints
+                # "endpoint": "URL of the deSEC API.", # TODO add documentation for this
```

Comment:
You added documentation in the README in this commit. What other docs do you mean?

Reply:
This TODO was meant to be gone...